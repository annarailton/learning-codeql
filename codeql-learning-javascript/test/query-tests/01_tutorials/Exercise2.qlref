01_tutorials/Exercise2.ql