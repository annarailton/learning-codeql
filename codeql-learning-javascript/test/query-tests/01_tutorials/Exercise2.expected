| -0.8900093488562771 |
